lib/game.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+require_relative 'board.rb'
+require_relative 'player.rb'
+require_relative 'ui.rb'
+
+class Game
+  attr_accessor :continue
+  def initialize(board, player1, player2, ui)
+    @new_board = board
+    @player1 = player1
+    @player2 = player2
+    @ui = ui
+    @ui.show_board(@new_board)
+    play
+  end
+
+  def play
```

Comment:
Ok, don't you think our `play` method is a bit overworked/doing too much? Probably some of the logic here can be delegated to private methods or even to our other classes?